docs/live_update.md

```diff
@@ -30,53 +30,43 @@ The `Tiltfile` at the root of the repo contains this example:
-  .add('./cmd/demoserver1',
-      '/go/src/github.com/windmilleng/tiltdemo/cmd/demoserver1')
-  .run('go install github.com/windmilleng/tiltdemo/cmd/demoserver1'))
+docker_build(dm1_img_name, '.', dockerfile='Dockerfile.server1')
+live_update(dm1_img_name,
+  [
+    sync('cmd/demoserver1', '/go/src/github.com/windmilleng/tiltdemo/cmd/demoserver1'),
+    run('go install github.com/windmilleng/tiltdemo/cmd/demoserver1'),
+    restart_container(),
+  ])
 ```
 
-This looks similar to the `Tiltfile` in previous tutorials, but instead of building
-with `docker_build()`, it contains `fast_build()`. Let's zoom
-in on that part of the function.
+This looks similar to the `Tiltfile` in previous tutorials, but in addition to building
+with `docker_build()`, it contains `live_update()`. Let's zoom
+in on that part of the configuration.
 
 
 ```python
-(fast_build(dm1_img_name, 'Dockerfile', '/go/bin/demoserver1')
-  .add('./cmd/demoserver1',
-      '/go/src/github.com/windmilleng/tiltdemo/cmd/demoserver1')
-  .run('go install github.com/windmilleng/tiltdemo/cmd/demoserver1'))
+live_update(dm1_img_name,
+  [
+    sync('cmd/demoserver1', '/go/src/github.com/windmilleng/tiltdemo/cmd/demoserver1'),
+    run('go install github.com/windmilleng/tiltdemo/cmd/demoserver1'),
+    restart_container(),
+  ])
 ```
 
 These lines configure `tilt` to do incremental image builds. We'll step through it line-by-line.
 
-* `fast_build(dm1_img_name, 'Dockerfile', '/go/bin/demoserver1')`
+* `live_update(dm1_img_name,`
 
-`fast_build` begins the build.
-This is setting up the build environment before we add any code.
-We build on top of the image in `Dockerfile`. Our new
-image has the name in `dm1_img_name` and has an entrypoint `/go/bin/demoserver1`.
+This specifies we're configuring live updates for any container running `dm1_img_name`.
 
-Here's what's in `Dockerfile`:
+* `sync('cmd/demoserver1', '/go/src/github.com/windmilleng/tiltdemo/cmd/demoserver1'),`
 
-```dockerfile
-FROM golang:1.10
-```
-
-It's only one line! This line says we're starting in a golang:1.10 container.
-
-Fast build Dockerfiles cannot contain any ADD or COPY lines.
-It's only for setting up the environment, not for adding your code.
-So this Dockerfile might look different than most.
-
-* `add('./cmd/demoserver1', '/go/src/github.com/windmilleng/tiltdemo/cmd/demoserver1')`
+The `sync` method copies a file or directory from outside your container to inside of your container.
 
-The `add` method copies a directory from outside your container to inside of your container.
-
-In this case, we copy the directory `./cmd/demoserver` (relative to the Tiltfile) into
+In this case, we copy the directory `./cmd/demoserver1` (relative to the Tiltfile) into
 the container filesystem.
 
-While Tilt is running, it watches all files in `./cmd/demoserver`. If they change, it copies the file
+While Tilt is running, it watches all files in `./cmd/demoserver1`. If they change, it copies the file
```

Comment:
not quite -- it watches all the files in `.`, but if they match `./cmd/demoserver1`, it runs the liveupdate steps, otherwise does a full build. put in some more here?